### PR TITLE
Add dotenv-rails gem to fix environment variable error for db setup

### PR DESCRIPTION
**Pull request by kiliczsh**
Environment variables were not read by db setup command. 

```bash
rails db:setup --trace
** Invoke db:setup (first_time)
** Invoke db:create (first_time)
** Invoke db:load_config (first_time)
** Invoke environment (first_time)
** Execute environment
** Execute db:load_config
** Execute db:create
connection to server at "127.0.0.1", port 5432 failed: fe_sendauth: no password supplied
Couldn't create 'maybe_development' database. Please check your configuration.
bin/rails aborted!
ActiveRecord::ConnectionNotEstablished: connection to server at "127.0.0.1", port 5432 failed: fe_sendauth: no password supplied (ActiveRecord::ConnectionNotEstablished)
```
@Shpigford

### CODE REVIEW

**Comment by pmareke** [comment]
> Environment variables were not read by db setup command.
> 
> ```shell
> rails db:setup --trace
> ** Invoke db:setup (first_time)
> ** Invoke db:create (first_time)
> ** Invoke db:load_config (first_time)
> ** Invoke environment (first_time)
> ** Execute environment
> ** Execute db:load_config
> ** Execute db:create
> connection to server at "127.0.0.1", port 5432 failed: fe_sendauth: no password supplied
> Couldn't create 'maybe_development' database. Please check your configuration.
> bin/rails aborted!
> ActiveRecord::ConnectionNotEstablished: connection to server at "127.0.0.1", port 5432 failed: fe_sendauth: no password supplied (ActiveRecord::ConnectionNotEstablished)
> ```
> 
> @Shpigford

I've just reported this error, thanks so much!

**Comment by kiliczsh** [comment]
> > Environment variables were not read by db setup command.
> > ```shell
> > rails db:setup --trace
> > ** Invoke db:setup (first_time)
> > ** Invoke db:create (first_time)
> > ** Invoke db:load_config (first_time)
> > ** Invoke environment (first_time)
> > ** Execute environment
> > ** Execute db:load_config
> > ** Execute db:create
> > connection to server at "127.0.0.1", port 5432 failed: fe_sendauth: no password supplied
> > Couldn't create 'maybe_development' database. Please check your configuration.
> > bin/rails aborted!
> > ActiveRecord::ConnectionNotEstablished: connection to server at "127.0.0.1", port 5432 failed: fe_sendauth: no password supplied (ActiveRecord::ConnectionNotEstablished)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > @Shpigford
> 
> I've just reported this error, thanks so much!

Resolves #335

File: .env.example [hunk 1]
```diff
@@ -12,4 +12,9 @@ SMTP_ADDRESS=
 SMTP_PORT=465
 SMTP_USERNAME=
 SMTP_PASSWORD=
-TLS=true
\ No newline at end of file
+TLS=true
+
+# Database Configuration
```
Comment (pmareke) [inline .env.example:17]: Should we add in the README the default values in the .env?

**Review by pmareke — COMMENTED** [review]
Thanks so much for the PR!

File: .env.example (hunk 1 above)
Comment (kiliczsh) [reply .env.example:17]: @pmareke it would be better if we add default values to .env.example

**Comment by lorenzopalaia** [comment]
Pointed out this issue on Discord earlier. I can confirm that this fixes it.

File: .env.example (hunk 1 above)
Comment (kiliczsh) [reply .env.example:17]: ✅ 🚀

**Pull request merged**